### PR TITLE
Use node executor and parameterise the version

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ parameters:
...
+    default: 14.17-browsers
+
 executors:
-  builder:
-    docker:
-      - image: 'cimg/node:14.17-browsers'
-    working_directory: ~/app
   integration-tests:
     docker:
       - image: cimg/node:14.17-browsers
```
Comment (andrewrlee) [inline .circleci/config.yml:19]: can use the parameter here too

Comment (petergphillips) [inline .circleci/config.yml:19]: would also be good to reference `node-version` here

**Review by petergphillips — DISMISSED** [review]
LGTM

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,20 +9,22 @@ parameters:
...
+    type: string
+    default: 14.17-browsers
+
 executors:
-  builder:
-    docker:
-      - image: 'cimg/node:14.17-browsers'
-    working_directory: ~/app
   integration-tests:
     docker:
```
Comment (petergphillips) [inline .circleci/config.yml:18]: think you can now change this to use the `node_redis` executor from hmpps too :grin:

Comment (markreesmoj) [reply .circleci/config.yml:18]: Ha I saw that PR coming and that's why I didn't bother to parameterise this section in the first place! One more push

**Review by andrewrlee — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 3]
```diff
@@ -92,7 +91,10 @@ jobs:
           path: test-results/unit-test-reports.html
 
   integration_test:
-    executor: integration-tests
+    executor:
+      name: hmpps/node_redis
+      node_tag: << pipeline.parameters.node-version >>
+      redis_tag: buster
```
Comment (petergphillips) [inline .circleci/config.yml:97]: think this should probably be `4-buster` to match the elasticache image, but was already set incorrectly so :shrug:

**Pull request merged**